q2_fmt/_examples.py

```diff
@@ -110,9 +115,22 @@ def wilcoxon_baseline0(use):
 
     stats_table.assert_output_type('StatsTable[Pairwise]')
 
-#TODO: usage example for mann-whitney
-def mann_whitney_u(use):
-    pass
+def mann_whitney_pairwise(use):
+    refdist = use.init_artifact('refdist', faithpd_refdist_factory)
+
+    stats_table, = use.action(
+        use.UsageAction('fmt', 'mann_whitney_u'),
+        use.UsageInputs(
+            distribution=refdist,
+            hypothesis='all-pairwise',
+            p_val_approx='asymptotic',
```

Comment:
I would add the `against_each` param, since that'll be the most useful version of this I think.